### PR TITLE
Implement Testing for App

### PR DESCRIPTION
**Pull request by hienphan0111**
- [ ] Set up React Testing Library.
- [ ] Write unit tests for the files operate.js and calculate.js using Jest.
- [ ] Create unit tests for all React components:
- Use Jest snapshots to test the components.
- Use React Testing Library to simulate user interaction.
- [ ] Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: src/logic/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import calculate from "./calculate";
```
Comment (FrederickMih) [inline src/logic/calculate.test.js:1]: - [ ] Based on the requirements, all functionalities and scenarios of the calculate.js module should be covered using jest. Let's kindly make sure the following scenarios are equally tested:
- (buttonName === '.') 
- (buttonName === '+/-')

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @hienphan0111, @amklass11, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights :green_circle: 

- PR is professional :+1: 
- All unit tests are passing :+1: 
- App still works on the browser :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

- The MIT license link on your README should be present and working. In order to render our README professional, let's kindly fix the issue.

_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/__snapshots__/Calculator.test.jsx.snap [hunk 2]
```diff
@@ -0,0 +1,204 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Calculator component Matches DOM Snapshot 1`] = `
```
Comment (BenMukebo) [inline src/components/__snapshots__/Calculator.test.jsx.snap:3]: - [OPTIONAL] According to the project requirements, There should be the usage of Jest snapshots for testing all the components. however, I only see the Calculator and Route test snapshots 🤔 Please make sure to add Navbar Jest snapshots test ✍🏻

**Review by BenMukebo — APPROVED** [review]
### Hi @TEAM, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
-  All the functionalities & scenarios of the `operate.js` and `calculate.js` is covered using Jest. :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**